english/adipose/adipose.md

```diff
@@ -15,4 +15,11 @@ I must resolve this marshmallow dream.
 
 [Wake Up](awake/awake.md)
 
-You realize you can't wake up. Or you are already awake. But you must continue because
\ No newline at end of file
+You realize you can't wake up. Or you are already awake. But you must continue because
+
+winter is coming.  
+
+Hot cocoa will be sold at nearly every street corner, and your sweet scent will likely have the drinkers plucking pieces of your being for their drinks.  Sure, it's not right as it's not consensual, but you have a hazy memory of a long ago cold night where you were running after having just passed a cocoa house.  The people all started following you with their warm, steamy cups held fully out as they chanted "mellows, mellows".  It eirily reminded you of a story you read as a young fluff, but the chant was, "brains, brains".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.